package.json

```diff
@@ -16,7 +16,7 @@
   "homepage": "https://protofire.github.io/solhint/",
   "repository": "https://github.com/protofire/solhint",
   "scripts": {
-    "exec-tests": "nyc -x \"**/grammar/**\" mocha && nyc report --reporter=text-lcov | coveralls",
+    "exec-tests": "nyc -x \"**/grammar/**\" mocha --recursive \"./test/rules/**/*.js\" && nyc report --reporter=text-lcov | coveralls",
```

Comment:
Why is this necessary?

Reply:
Mocha searches for the files in the test directory. Mocha does not find the tests in the subfolder rules. I think with the recursive option it would be fine